test/mocks/StakingRewardsMock.sol

```diff
@@ -0,0 +1,212 @@
+/**
+ * NOTICE: This contract has been adapted from the original Synthetix source code:
+ *  - Upgrade Solidity version from 0.5.x to 0.8.x.
+ *  - Add referral code functionality for `stake()`.
+ *
+ * Original: https://github.com/Synthetixio/synthetix/blob/5e9096ac4aea6c4249828f1e8b95e3fb9be231f8/contracts/StakingRewards.sol
+ *     Diff: https://www.diffchecker.com/rgBMl45S/
+ */
+
```

Comment:
```suggestion
```
I'd remove this as not really important for a mock, also because the diff is old and nor appropriate for it, as it is not including the specific mock changes.